containers/podman-desktop
# feat: add initial support for contexts

### What does this PR do?

This adds an initial support to work with contexts.
It has been extracted from https://github.com/containers/podman-desktop/pull/3131 to allow a quicker and easier review/merge.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

it is part of https://github.com/containers/podman-desktop/pull/3131

### How to test this PR?

N/A

@benoitf this also contain the fixes based on your comments in https://github.com/containers/podman-desktop/pull/3131

Still not sure what text i should add to mark that some code has been taken from the Vscode project
